build.gradle

```diff
@@ -8,8 +15,8 @@ apply plugin: 'eclipse'
 apply plugin: 'idea'
 apply plugin: 'checkstyle'
 apply plugin: 'pmd'
-apply plugin: 'findbugs'
-apply plugin: 'maven-publish'
+apply plugin: 'com.github.spotbugs'
+//apply plugin: 'maven-publish'
```

Comment:
What is the problem/matter with this plugin?

Reply:
Looks like this was just an outdated snippet